salt/modules/influxdbmod.py

```diff
@@ -407,6 +407,7 @@ def create_retention_policy(database,
                             duration,
                             replication,
                             default=False,
+                            shard_duration="0s",
```

Comment:
since these are new features for these modules the commits ded8f4c and 3a2eb54 should go into develop. Also can you add some tests for both the features and the bug fix? thanks :)

Reply:
I can move the shard_duration commits to the develop branch if required, but I'm sorry I won't be able to write tests since I'm currently not familiar with testing in saltstack and no test already exist for the influxdb module and states (and I won't have the bandwidth currently to dive in testing and write the tests for influxdb stuff).